ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -0,0 +1,78 @@
+@page

```

Comment:
Perhaps, as it's throw-away UI, we should reflect it with where it sits in the page structure: `/Pages/TempLegacySearch/Index`? I'm suspecting we might want to keep this in parallel to starting actual search page dev, plus it makes it clear that it's not final in any way